docs/proposals/volumes-proposal.md

```diff
@@ -38,23 +46,29 @@ In above example, we have services `db`, `backup` and `process`.
 - `process` is a more of a job which is dataprocessing application, which does not need persistent storage but needs storage until it does it's job.
 
 
-Conversion of root level `volumes` directive:
+### Root level `volumes` directive:
 
-- We generate `pvc` for each entry in the volumes directive.
+We generate `pvc` for each entry in the volumes directive.
 
-  `access` -> `pvc.spec.accessModes`
+- `name`:          (required field, type: *string*) name of volume, which is referenced from containers for mounting. `pvc` will be created with this name.
+- `size`:          (required field, type: *string*) In `pvc` this will be put in `pvc.spec.resources.requests`
+- `accessMode`:    (optional field, type: *string*) In `pvc` this will turn into `pvc.spec.accessModes`. The possible values of this field could be: `ReadOnlyMany`, `ReadWriteOnce` or `ReadWriteMany`. **TODO**: Decide default value because `pvc.spec.accessModes` is a required field.
+- `storageClass`:  (optional field) In `pvc` this will be into `pvc.metadata.annotations` as
+  ```json
+  "annotations": {
+    "volume.beta.kubernetes.io/storage-class": "user-value"
+  }
+  ```
+  src: http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html
+- `reclaimPolicy`: (optional field, type: *string*) **TODO**: No field in `pvc`. 
```

Comment:
oh my bad :blush: 
we  shouldn't have `reclaimPolicy`  here. This is PersistentVolume property, so we don't have to worry about that.